### PR TITLE
New package: YmSoft.Nalgaeset version 10.40.83

### PR DESCRIPTION
**Pull request by dahlia**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

> ### [Nalgaeset Hangul Input System](http://moogi.new21.org/en/ngs/)
>
> _Nalgaeset_ is a Korean input method software for Microsoft Windows. It mainly consists of both a self-contained text editor and a universal IME for the operating system. The two programs share the common library, and support virtually every Windows platform from 95/NT4 to 10 of any languages, both 32- and 64-bit.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/46092)

### CODE REVIEW

File: manifests/y/YmSoft/Nalgaeset/10.4/YmSoft.Nalgaeset.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+PackageIdentifier: YmSoft.Nalgaeset
+PackageVersion: 10.4
+PackageLocale: en-US
+Publisher: YmSoft
+PublisherUrl: http://moogi.new21.org/en/
+Author: Hong Minhee
+PackageName: Nalgaeset Hangul Input System
```
Comment (ItzLevvie) [inline manifests/y/YmSoft/Nalgaeset/10.4/YmSoft.Nalgaeset.locale.en-US.yaml:7]:
```suggestion
PackageName: 날개셋 한글 입력기
```
shows up as `날개셋 한글 입력기` for en-US devices.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/y/YmSoft/Nalgaeset/10.4/YmSoft.Nalgaeset.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+PackageIdentifier: YmSoft.Nalgaeset
+PackageVersion: 10.4
+Installers:
```
Comment (ItzLevvie) [inline manifests/y/YmSoft/Nalgaeset/10.4/YmSoft.Nalgaeset.installer.yaml:3]:
```suggestion
Scope: machine
Installers:
```
`C:\Program Files\YmSoft\NgsEdit`

**Comment by dahlia** [comment]
@ItzLevvie Thank you for reviewing!  I adjusted points you addressed.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**